graylog2-web-interface/src/components/lookup-tables/CacheForm.jsx

```diff
@@ -184,17 +185,28 @@ class CacheForm extends React.Component {
     return <span>{defaultText}</span>;
   };
 
+  _renderTitle = (title, typeName, create) => {
+    const TagName = create ? 'h3' : 'h2';
+    return <TagName>{title} <small>({typeName && typeName})</small></TagName>;
+  };
+
   render() {
     const { cache } = this.state;
-    const { create, type } = this.props;
+    const {
+      create,
+      type,
+      title,
+    } = this.props;
 
     const cachePlugins = PluginStore.exports('lookupTableCaches');
 
     const plugin = cachePlugins.filter(p => p.type === type);
     let configFieldSet = null;
     let documentationComponent = null;
+    let pluginDisplayName = null;
```

Comment:
When a plugin cannot be found for some reason (it was uninstalled), I think we should display the raw type stored in the configuration `cache.config.type`. That would be consistent with other parts of the application and also shows something somewhat helpful in that case.

Reply:
Fixed